src/Atmos/Model/AtmosModel.jl

```diff
@@ -619,6 +619,9 @@ function transform_post_gradient_laplacian!(
     )
 end
 
+precompute(atmos::AtmosModel, args, ::Flux{SecondOrder}) =
+    (ts = recover_thermo_state(atmos, args.state, args.aux),)
```

Comment:
Wondering if you want to add: `ν, D_t, τ = turbulence_tensors(atmos, state, diffusive, aux, t)` to this.

Reply:
I thought the same thing, and I did at first, but then I thought that it might be PMO because `turbulence_tensors` is cheap AFAICT. It's an easy thing to change. I'll incorporate it in a follow-up PR, so we can compare the buildkite runtime to see if there's a noticeable gain.